src/book/mod.rs

```diff
@@ -519,6 +519,14 @@ impl MDBook {
         &[]
     }
 
+    pub fn uses_ace(&self) -> bool {
```

Comment:
`config` is a private property on an `MDBook`, which means I can't reach it from `hbs_renderer`, so I followed what seemed to be a convention in creating these extra methods in `MDBook` to pull out values from the private `config`. There does not appear to be a `get_config` function.

Am I missing something?